R/limits.r

```diff
@@ -184,10 +184,9 @@ limits.POSIXlt <- function(lims, var, call = caller_env()) {
 expand_limits <- function(...) {
   data <- list2(...)
   data_dfs <- vapply(data, is.data.frame, logical(1))
-  data <- do.call(c, c(list(data[!data_dfs]), data[data_dfs]))
+  data <- unlist(c(list(data[!data_dfs]), data[data_dfs]), recursive = FALSE)
```

Comment:
I don't know if it's important, but I have no intuition for what this code does. Is it `inject(c(data[!data_dfs], !!!data[data_dfs])`?

Reply:
Ah, it's `tidy::unpack()`; you might just mention "unpack data frame columns" as a comment. I don't think there's a function for this in vctrs yet, but @DavisVaughan can confirm.